plugin/key_column.go

```diff
@@ -122,7 +122,7 @@ func (k *KeyColumn) Validate() []string {
 	// first set default operator and convert "!=" to "<>"
 	k.InitialiseOperators()
 	// ensure operators are valid
-	validOperators := []string{"=", "<>", "<", "<=", ">", ">=", quals.QualOperatorIsNull, quals.QualOperatorIsNotNull}
+	validOperators := []string{"=", "<>", "<", "<=", ">", ">=", "~~", "!~~", "~~*", "!~~*", quals.QualOperatorIsNull, quals.QualOperatorIsNotNull}
```

Comment:
I'd like to turn this list into a public constant, allowing table implementations to refer to it.

WDYT?

Reply:
@pdecat I agree valid operators could be moved to a public const at the top of that file. It may also worth adding a  comment describing the added operators